cernopendata/modules/fixtures/data/records/data-policies.json

```diff
@@ -0,0 +1,120 @@
+[
+{
+  "abstract": {
+    "description": "This document contains the LHCb Data Access Policy. This was adopted at the Collaboration Board meeting on 27th Feb 2013."
+  }, 
+  "accelerator": "CERN LHC", 
+  "authors": [
+    {
+      "affiliation": "U. Edinburgh", 
+      "name": "Clarke, Peter"
+    }
+  ], 
+  "collaboration": {
+    "name": "LHCb collaboration"
+  }, 
+  "collections": [
+    "Data-Policies"
+  ], 
+  "date_published": "2013", 
+  "doi": "10.7483/OPENDATA.LHCb.HKJW.TWSZ", 
+  "experiment": "LHCb", 
+  "language": "eng", 
+  "prepublication": {
+    "date": "22 Apr 2013", 
```

Comment:
Should we use a standardised format for dates? e.g. ISO 8601
That's perhaps something for another time.

Reply:
We have only four values here, so we could do it now...

Reply:
YYYY-MM-DD should suffice

Reply:
Done. Example:

```diff
-    "date": "03 May 2012", 
+    "date": "2012-05-03", 
```